setup/config/config_pushserver.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+type PushServer struct {
+	Matrix *Global `yaml:"-"`
+
+	// PushGatewayTLSInsecureSkipVerify controls whether the Push
+	// gateway client should verify X.509 certificates, or allow
+	// anything. Only for testing, with e.g. self-signed certificates.
+	PushGatewayTLSInsecureSkipVerify bool `yaml:"push_gateway_tls_insecure_skip_verify_for_testing"`
```

Comment:
Please be consistent with other flags. See federation sender which has a similar option: https://github.com/matrix-org/sytest/pull/1172/files#diff-a50fea42e82ac587405319c40b1e270ccab6f6af970d1444bd547822330c5bd9R175